common/test/acceptance/pages/lms/video/video.py

```diff
@@ -356,9 +356,9 @@ def click_player_button(self, button):
 
         self.q(css=button_selector).first.click()
 
-        button_states = {'play': 'playing', 'pause': 'pause'}
-        if button in button_states:
-            self.wait_for_state(button_states[button])
+        # button_states = {'play': 'playing', 'pause': 'pause'}
+        # if button in button_states:
+        #     self.wait_for_state(button_states[button])
```

Comment:
@polesye with the current code in master, when the tests fail intermittently we see that the button state is is-buffered, no matter how long you wait.  I just did a quick look back through jenkins but unfo don't see any of those results currently (we delete old builds in order to save space).